mlos_bench/mlos_bench/config/services/remote/azure/scripts/setup-rg/setup-rg.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+##
+## Copyright (c) Microsoft Corporation.
+## Licensed under the MIT License.
+##
+
+set -eu
+
+# Argument parsing
+while [[ "$#" -gt 0 ]]; do
+case $1 in
+    # Main control plane ARM template params
+    --controlPlaneArmParameters)
+        controlPlaneArmParameters="$2"
+        shift 2
+        ;;
+    # Results DB ARM template params
+    --resultsDbArmParameters)
```

Comment:
```suggestion
    --resultsDbArmParametersFile)
```

See also comments from README